lib/gds_api/publishing_api/special_route_publisher.rb

```diff
@@ -12,7 +12,8 @@ def initialize(options = {})
       def publish(options)
         logger.info("Publishing #{options.fetch(:type)} route #{options.fetch(:base_path)}, routing to #{options.fetch(:rendering_app)}")
 
-        publishing_api.put_content_item(options.fetch(:base_path), {
+        put_content_response = publishing_api.put_content(options.fetch(:content_id), {
+          base_path: options.fetch(:base_path),
           content_id: options.fetch(:content_id),
```

Comment:
`content_id` is no longer required in V2 (because it's inferred from the URL params)